papaparse.js

```diff
@@ -850,9 +850,9 @@
 				{
 					_results.data[i] = row;
 					if (j > _fields.length)
-						addError("FieldMismatch", "TooManyFields", "Too many fields: expected " + _fields.length + " fields but parsed " + j, i);
+						addError("FieldMismatch", "TooManyFields", "Too many fields: expected " + _fields.length + " fields but parsed " + j, self.streamer._rowCount + i);
 					else if (j < _fields.length)
-						addError("FieldMismatch", "TooFewFields", "Too few fields: expected " + _fields.length + " fields but parsed " + j, i);
+						addError("FieldMismatch", "TooFewFields", "Too few fields: expected " + _fields.length + " fields but parsed " + j, self.streamer._rowCount + i);
```

Comment:
First i thought yes... but you mean when it's in row 0 of a chunk, right? I'm gonna check that.

@mholt @bluej100 Apart from that, what do you think of touching the main parser like that (for fixing MissingQuotes row error)? I still think it's not the best way, to do that. Can you think of some other to do it?